cli/src/main.rs

```diff
@@ -45,58 +76,118 @@ struct Connection {
 
 #[derive(Subcommand)]
 enum Commands {
-    Status,
-    Deploy(DeployArgs),
-}
+    // status
+    Status(status::StatusArgs), //aka topology
+
+    // deployment
+    DeployResource(deploy_resource::DeployResourceArgs),
+
+    // process instance
+    CreateProcessInstance(create_process_instance::CreateProcessInstanceArgs),
+    CancelProcessInstance(cancel_process_instance::CancelProcessInstanceArgs),
 
-#[derive(Args)]
-struct DeployArgs {
-    #[clap(required = true, value_parser, value_name = "FILE")]
-    resources: Vec<PathBuf>,
+    // message
+    PublishMessage(publish_message::PublishMessageArgs),
+
+    // incident
+    ResolveIncident(resolve_incident::ResolveIncidentArgs),
+
+    // variables
+    SetVariables(set_variables::SetVariablesArgs),
+
+    //jobs
+    ActivateJobs(activate_jobs::ActivateJobsArgs),
+    CompleteJob(complete_job::CompleteJobArgs),
+    FailJob(fail_job::FailJobArgs),
+    UpdateRetries(update_retries::UpdateRetriesArgs),
+    ThrowError(throw_error::ThrowErrorArgs),
 }
 
 impl From<Connection> for zeebe_client::Connection {
     fn from(conn: Connection) -> Self {
         match conn.address {
-            Some(addr) => zeebe_client::Connection::Address(addr),
-            None => zeebe_client::Connection::HostPort(conn.host, conn.port),
+            Some(addr) => zeebe_client::Connection::Address {
+                insecure: conn.insecure,
+                addr,
+            },
+            None => zeebe_client::Connection::HostPort {
```

Comment:
:question: Do we want to keep these two ways? Isn't the Zeebe Address the canonical form and we could convert `HostPort` into `Address`?

Reply:
Good point, it'd be nice if the client only accepts an Uri :+1: